fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/DefaultRequest.kt

```diff
@@ -192,11 +192,13 @@ data class DefaultRequest(
      * @param openStream [BodySource] a function that yields a stream
      * @param calculateLength [Number?] size in +bytes+ if it is known
      * @param charset [Charset] the charset to write with
+     * @param repeatable [Boolean] loads the body into memory upon reading
      *
      * @return [Request] the request
      */
-    override fun body(openStream: BodySource, calculateLength: BodyLength?, charset: Charset): Request {
+    override fun body(openStream: BodySource, calculateLength: BodyLength?, charset: Charset, repeatable: Boolean): Request {
         _body = DefaultBody.from(openStream = openStream, calculateLength = calculateLength, charset = charset)
+                           .let { body -> if (repeatable) body.asRepeatable() else body }
         return request
```

Comment:
**[ktlint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Unexpected indentation (27) (it should be 12)